tests/utils/esx/vim_cmd.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// vim-cmd helpers
+
+package esx
+
+import (
+	"fmt"
+
+	"github.com/vmware/docker-volume-vsphere/tests/constants/esx"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
+)
+
+// SuspendResumeVM - suspend and resume a VM identified by name
```

Comment:
I don't have much idea about vim-cmd. can you help me to understand for not using GOVC approach?